application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadAutoArchiver.java

```diff
@@ -28,8 +27,8 @@
  */
 public final class HelpThreadAutoArchiver implements Routine {
     private static final Logger logger = LoggerFactory.getLogger(HelpThreadAutoArchiver.class);
-    private static final int SCHEDULE_MINUTES = 60;
-    private static final Duration ARCHIVE_AFTER_INACTIVITY_OF = Duration.ofHours(12);
+    private static final int SCHEDULE_MINUTES = 9;
```

Comment:
debugging leftovers? also line 31 and 46

Reply:
it's an outdated commit.

Reply:
lmao